docs/get-details/indexer.md

```diff
@@ -899,7 +903,235 @@ Results
 }
 ```
 
-The ledger data will include both Algo balance and if the account has any assets with their associated balances. This call supports [Historical Data Searches](#historical-data-searches) with the `round` parameter which will return ledger data for the account at a specific round.
+The ledger data will include the Algo balance and any extra account information such as `created-apps`, `created-assets`, `apps-local-state`, and `assets` by default. This can be excluded using the `exclude` parameter. This call supports [Historical Data Searches](#historical-data-searches) with the `round` parameter which will return ledger data for the account at a specific round.
+
+# Lookup applications created by a specific account
+The `/accounts/{account-id}/created-apps` REST call allows for searching an account for all the applications it has created.
+
+=== "JavaScript"
+    ```javascript
+    (async () => {
+        let address = "XIU7HGGAJ3QOTATPDSIIHPFVKMICXKHMOR2FJKHTVLII4FAOA3CYZQDLG4";
+        let response = await indexerClient.lookupAccountCreatedApplications(address).do();
+        console.log(JSON.stringify(response, undefined, 2));
+        }
```

Comment:
nit: can you format this? usually the closing curly would be indented 1 tab less than the content and often the closing paren is immeditely adjacent.

Also instead of console.log in the catch case, a console.error(...)